lib/commanded/aggregate.ex

```diff
@@ -38,15 +38,18 @@ defmodule OpentelemetryCommanded.Aggregate do
     :otel_propagator_text_map.extract(trace_headers)
 
     attributes = [
-      "command.type": struct_name(context.command),
-      "command.handler": context.handler,
-      "aggregate.uuid": meta.aggregate_uuid,
-      "aggregate.version": meta.aggregate_version,
-      application: meta.application,
-      "causation.id": context.causation_id,
-      "correlation.id": context.correlation_id,
-      "aggregate.function": context.function,
```

Comment:
do we still want these two fields?

Reply:
`aggregate.function": context.function` is still there, just massaged to fit naming for messaging attributes:
```elixir
      "messaging.commanded.function": context.function
```

I can restore `context.lifespan` but while testing, didn't see it's value as an attribute since it doesn't effect the processing of the current span.  Thoughts?